apps/web/src/pages/TalentNominations/NominateTalent/components/SubHeading.tsx

```diff
@@ -1,13 +1,33 @@
-import { Heading, HeadingProps } from "@gc-digital-talent/ui";
-
-const SubHeading = (props: HeadingProps) => (
-  <Heading
-    level="h2"
-    color="primary"
-    data-h2-margin-top="base(0)"
-    data-h2-font-weight="base(400)"
-    {...props}
-  />
-);
+import { useEffect, useRef } from "react";
+
+import { Heading, HeadingProps, HeadingRef } from "@gc-digital-talent/ui";
+
+interface SubHeadingProps extends HeadingProps {
+  preventAutoFocus?: boolean;
+}
+
+const SubHeading = ({ preventAutoFocus, ...rest }: SubHeadingProps) => {
+  const headingRef = useRef<HeadingRef>(null);
+
+  useEffect(() => {
+    if (headingRef.current && !preventAutoFocus) {
+      // Focus heading and scroll to top
+      headingRef.current.focus({ preventScroll: false });
+      window.scrollTo({ top: 0, left: 0, behavior: "instant" });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
What's the reason to omit `preventAutoFocus` from the dependency array?

Reply:
I want it to run on mount which needs an empty dependency array.